src/mysqlsh_helpers.py

```diff
@@ -46,15 +71,36 @@ def mysqlsh_common_dir(self) -> str:
         Returns:
             Path to common dir
         """
-        return "/root/snap/mysql-shell/common"
+        if os.path.exists("/root/snap/mysql-shell/common"):
+            return "/root/snap/mysql-shell/common"
+        else:
+            return "/tmp"
 
     def configure_mysql_users(self):
         """Configure the MySQL users for the instance.
 
-        Creates a 'clusteradmin' user with the appropriate privileges and
-        revokes certain privileges from the 'root' user.
+        Creates base `root@%` and `clusteradmin@instance_address` user with the
+        appropriate privileges, and reconfigure `root@localhost` user password.
         """
-        pass
+        _script = (
+            "SET @@SESSION.SQL_LOG_BIN=0;",
```

Comment:
why do we need to disable bin log here? may be add a comment if this is necessary

Reply:
This should not be replicated anywhere as it can harm other apps access. Will add a comment.